lib/blue_bird/conn_logger.ex

```diff
@@ -75,8 +75,10 @@ defmodule BlueBird.ConnLogger do
       iex> save(conn)
       :ok
   """
-  @spec save(Plug.Conn.t()) :: :ok
-  def save(conn) do
+  @type save_option :: {:title, String.t()}
```

Comment:
Inspired by https://elixirforum.com/t/typespecs-best-way-to-spec-keyword-lists/2991/3?u=elvanja